Berksfile

```diff
@@ -1,3 +1,5 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+cookbook 'windows-security-policy', git: 'https://github.com/grdnrio/windows-security-policy.git'
```

Comment:
Once this cookbook is available in supermarket, we do not need this here anymore